.flake8

```diff
@@ -0,0 +1,15 @@
+[flake8]
+ignore = 
+    E731,
```

Comment:
I know this is documented in the HOWTO, but it might be nice to have a comment here so that people reading this file have a quick way to see what these rules mean.